code/modules/wiremod/core/integrated_circuit.dm

```diff
@@ -626,6 +631,11 @@ GLOBAL_LIST_EMPTY_TYPED(integrated_circuits, /obj/item/integrated_circuit)
 
 #undef WITHIN_RANGE
 
+/obj/item/integrated_circuit/balloon_alert_perform(mob/viewer, text)
```

Comment:
Why are you overriding the internal-just-for-INVOKE_ASYNC-only `balloon_alert_perform` and not just `balloon_alert` itself?

Reply:
was probably tired at the time I made this